libs/api-bundle/tests/RequestMapper/RequestData.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ApiBundle\Tests\RequestMapper;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class RequestData
+{
+    public function __construct(
+        #[Assert\Length(min: 1, max: 16)]
+        public readonly string $name,
```

Comment:
> muzes dat nejaky link, kde se ti to deje? Idealne primo na nejaky test, abych si to mohl pustit.

Jsem si to vcera premazal :( 
Ale treba tady kdyz pridas Datetime a neposles ho, tak pak tu chybu dostanes taky.

```suggestion
        public readonly string $name,
        public readonly DateTimeImmutable $created,
```

Reply:
mrknu